turtlesim/Pose pose
---
string result
